### PR TITLE
Add options parameters from comment

### PR DESCRIPTION
**Pull request by shaedrich**
As per https://www.php.net/manual/en/mysqli.debug.php#122219

### CODE REVIEW

File: reference/mysqli/mysqli/debug.xml [hunk 1]
```diff
@@ -34,6 +34,62 @@
       <para>
        A string representing the debugging operation to perform
       </para>
+      <para>
+       <table>
+        <title>The following characters are recognized in the
+        <parameter>options</parameter> parameter string</title>
```
Comment (cmb69) [inline reference/mysqli/mysqli/debug.xml:40]: This title appears to be too verbose. Maybe just:
```suggestion
        <title>Recognized Flag Characters</title>
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Apparently, [some flags are missing](https://github.com/php/php-src/blob/php-7.4.25/ext/mysqlnd/mysqlnd_debug.c#L494-L688), but it's better than having no information here at all. And it might be worthwhile to also explain the [format generally](http://tiebing.blogspot.com/2011/10/cc-dbug-library.html), but again, better something than nothing.

**Comment by shaedrich** [comment]
@cmb69 You are welcome! I added an explanation and a missing flag – as you say, it's still not every information, but we already got closer to the goal

File: reference/mysqli/mysqli/debug.xml [hunk 2]
```diff
@@ -34,6 +34,71 @@
       <para>
        A string representing the debugging operation to perform
       </para>
+      <para>
+        The debug control string is a sequence of colon separated fields as follows:
+        <literal><![CDATA[<field_1>:<field_2>:<field_N>]]></literal>
+        Each field consists of a mandatory flag character followed by an optional "," and comma separated list of modifiers:
```
Comment (cmb69) [inline reference/mysqli/mysqli/debug.xml:40]:
```suggestion
        Each field consists of a mandatory flag character followed by an optional <literal>,</literal> and comma separated list of modifiers:
```

File: reference/mysqli/mysqli/debug.xml [hunk 3]
```diff
@@ -34,6 +34,71 @@
...
+        <literal>flag[,modifier,modifier,...,modifier]</literal>
+      </para>
+      <para>
+       <table>
+        <title>Recognized Flag Characters</title>
+          <tgroup cols="3">
```
Comment (cmb69) [inline reference/mysqli/mysqli/debug.xml:46]: Shouldn't this be:
```suggestion
          <tgroup cols="2">
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Looks good to me, but please check-out my comments.

File: reference/mysqli/mysqli/debug.xml (hunk 3 above)
Comment (shaedrich) [reply reference/mysqli/mysqli/debug.xml:46]: You're right :+1: I copied and pasted it and changed everything but this to two cols 😅

**Comment by shaedrich** [comment]
You're welcome!

**Pull request merged**